README.md

```diff
@@ -1 +1,47 @@
-Coming soon
+# PostgreSQL Kubernetes Operator
+
+## Overview
+
+This repository hosts the PostgreSQL Kubernetes Operator.
+
+## Description
+
+The PostgreSQL Kubernetes Operator deploys and operates the [PostgreSQL](https://www.postgresql.org/about/) database on Kubernetes clusters.
+
+This operator provides a Postgres database with replication enabled (one master instance and one or more hot standby replicas). The Operator in this repository is a Python script which wraps the LTS Postgres versions distributed by [Ubuntu](https://hub.docker.com/r/ubuntu/postgres), providing lifecycle management and handling events (install, configure, integrate, remove).
+
+## Usage
+
+In order to install this charm, execute the following commands (replace `SECRET_PASSWORD` with a strong password):
+
+```bash
+# Clone this charm respository
+$ git clone https://github.com/canonical/postgresql-k8s-operator
+$ cd postgresql-k8s-operator
+# Build the charm
+$ charmcraft pack
+# Create a model for our deployment
+$ juju add-model postgres
+# Deploy the charm (providing a password for the postgres user)
+$ juju deploy ./postgresql-k8s-operator*.charm \
+    --resource postgresql-image=ubuntu/postgres \
+    --config postgres-password=SECRET_PASSWORD
```

Comment:
In general, I'd suggest having the charm *generate* any required password like this, but again that can follow in the next PR.

Reply:
Thanks for the suggestion, Jon. I saw something similar in https://github.com/canonical/mongodb-operator/blob/4090146e11d8d05e5c05128a39346d3781144cae/src/charm.py#L273 and I'll implement in the next PR.